packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.js

```diff
@@ -28,6 +28,16 @@ import BpkScrollableCalendarGrid from './BpkScrollableCalendarGrid';
 
 const getClassName = cssModules(STYLES);
 
+const getMonthsArray = (startDate, count) => {
+  const months = [];
+
+  for (let i = 0; i < count + 1; i += 1) {
+    months.push(DateUtils.addMonths(startDate, i));
+  }
+
```

Comment:
`for` loops aren't hipster enough, how about:

```
const months = Array(count).fill().map((_, index) => DateUtils.addMonths(startDate, index));
```

Reply:
That's kinda how it looked before, but Internet Explorer 😢

Reply:
ah right